tpgtools/templates/provider_dcl_client_creation.go.tmpl

```diff
@@ -46,15 +46,15 @@ func NewDCL{{$pkg.ProductType}}Client(config *Config, userAgent, billingProject
 			dclClientOptions,
 			dclUserAgentOptions,
 			dclLoggerOptions,
-			dcl.WithBasePath(config.{{$pkg.ProductType}}BasePath),
+			dcl.WithBasePath(config.{{$pkg.BasePathName}}BasePath),
```

Comment:
perhaps rephrase `BasePathName` to `BasePathId` or `BasePathIdentifier` ? 

hmm im not sure name sounds weird to me

Reply:
only nit, feel free to ignore if you disagree